TF_Service_dotNet/TouchFreeTests/PositionTrackers/WristTrackerTests.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Numerics;
+using NUnit.Framework;
+using Ultraleap.TouchFree.Library.Interactions.PositionTrackers;
+
+namespace TouchFreeTests.PositionTrackers
+{
+    public class WristTrackerTests
+    {
+        WristTracker sut;
+
+        public WristTrackerTests()
+        {
+            sut = new WristTracker();
+        }
+
+        [TestCase(1000, 2000, 3000, 1, 2, 3)]
```

Comment:
Can these only take built-in types or can they take any object? I'm thinking if these are a pair of vector3's it'd be nice to capture them in that form for ease of reading.